editor/plugins/texture_editor_plugin.cpp

```diff
@@ -132,8 +132,19 @@ static Image::Format get_texture_2d_format(const Ref<Texture2D> &p_texture) {
 
 static int get_texture_mipmaps_count(const Ref<Texture2D> &p_texture) {
 	ERR_FAIL_COND_V(p_texture.is_null(), -1);
+
 	// We are having to download the image only to get its mipmaps count. It would be nice if we didn't have to.
-	Ref<Image> image = p_texture->get_image();
+	Ref<Image> image;
+	if (Ref<AtlasTexture> at = p_texture; at.is_valid()) {
```

Comment:
Assignment inside conditions is not allowed as per code style.

Reply:
Corrected, thanks.